src/rgw/rgw_rados.cc

```diff
@@ -1136,6 +1136,8 @@ int RGWRados::register_to_service_map(const string& daemon_type, const map<strin
   if (name.compare(0, 4, "rgw.") == 0) {
     name = name.substr(4);
   }
+  std::string instance_id = stringify(rados->get_instance_id());
```

Comment:
what provides stringify?

Reply:
Not sure, I took this from the rbd-mirror code 😅